### DISCUSSION TITLE
pass some data to update()

### DISCUSSION THREAD

**Discussion by briggeml** [hecrj/iced#1102] [Q&A] [answered]
I sorry in advance, I'm a newbie to Rust and Iced. But how to pass the custom data from inside`fn main()` received before call `CustomWindow::run(Settings::default())` to be used in the `fn update()` later ?

**Comment by hecrj** [comment, marked as answer]
Check out the answer of #798!

**Reply by briggeml** [reply]
Thanks, works for me!
